### ISSUE TITLE
wiki : Simple Password Authentication is correct?

### ISSUE THREAD

**Issue by no1-knows** [Sorcery/sorcery#205]
Hello, is this link_to code correct?

```
# app/views/user_sessions/new.html.erb
<h1>Login</h1>

<%= render 'form' %>

<%= link_to 'Back', user_sessions_path %>
```

This back link get "Routing Error No route matches [GET] “/user_sessions”".
I think users_path is better.

**Comment by mladenilic** [comment]
Yes, however tutorial assumes that you have defined `user_sessions` resources in your routes file and that you have created `UserSessionsController` controller file accordingly.

Keep in mind that Sorcery does not expect nor enforce any particular route setup – this is left entirely to the developer to decide.
